### PR TITLE
add publishing data for 15.8-preview1 branch

### PR DESCRIPTION
**Pull request by brettfo**

### CODE REVIEW

File: build/config/PublishData.json [hunk 1]
```diff
@@ -27,14 +27,21 @@
...
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.7p1"]
-        },     
+        },
+        "dev15.8-preview1-vs-deps": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.9.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev15.8p1"]
```
Comment (jaredpar) [inline build/config/PublishData.json]: you need to delete the existing dev15.8p1 tag which is on line 8

Comment (brettfo) [reply build/config/PublishData.json]: Thanks, fixed.

**Comment by brettfo** [comment]
@dotnet-bot test windows_debug_spanish_unit32_prtest `// not sure how this could have broken Spanish`

**Comment by brettfo** [comment]
@dotnet-bot test windows_debug_spanish_unit32_prtest please

**Comment by jaredpar** [comment]
@brettfo what was the spanish failure?

**Comment by brettfo** [comment]
Unit test failure: https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_debug_spanish_unit32_prtest/317/

**Comment by jaredpar** [comment]
Thanks. That is the third time I've seen the VB EE tests crash on spanish now. At first I was hoping it was a sample size problem and the tests were flaky and would also crash on English. But looking less likely now.

**Comment by jasonmalinowski** [comment]
@dotnet-bot retest windows_debug_spanish_unit32_prtest please.

**Pull request merged**